src/casl/casl-ability.factory.ts

```diff
@@ -163,7 +163,7 @@ export class CaslAbilityFactory {
     //  can(Action.Manage, Instrument);
     //}
 
-    can(Action.Manage, Job);
+    can(Action.Manage, JobClass);
```

Comment:
You do not need to change the casl authorization settings for jobs.
They have already been updated in the master branch according to the documentation:
https://github.com/SciCatProject/scicat-backend-next/blob/e3248e753fa5a01b1afb908a46db4f10e4e806e5/src/casl/casl-ability.factory.ts#L777

Reply:
Ok, I guess this needs to be merged into release-jobs